File: pegasusX/apps/supplier-app-android/app/src/main/java/com/pegasusx/supplier/data/remote/SupplierSessionReconcile.kt
package com.pegasusx.supplier.data.remote

/** Refetch server-authoritative supplier snapshots after transport reconnect. */
suspend fun reconcileSupplierSession(api: SupplierApi) {
    runCatching { api.getDispatchPreview() }
    runCatching { api.getManifests() }
}

File: pegasusX/apps/supplier-app-ios/SupplierApp/Services/SupplierSessionReconcile.swift
//
//  SupplierSessionReconcile.swift
//  SupplierApp
//
//  Refetch server-authoritative supplier snapshots after transport reconnect.
//

import Foundation

enum SupplierSessionReconcile {
    static func run() async {
        _ = try? await SupplierOperationsService.dispatchPreview()
        _ = try? await SupplierOperationsService.manifests()
    }
}
